torch/_inductor/utils.py

```diff
@@ -357,34 +360,79 @@ def aggregate_origins(node_schedule):
         return set()
 
 
-def get_fused_kernel_name(node_schedule, descriptive_names):
+def get_module_name_from_meta(meta_dict):
```

Comment:
reminder to myself, will take a closer look at this logic later

Reply:
Done - I added several comments and made the regex simpler.

eg.   
       Before: 'getattr_L__self___layer4___0___downsample_1'
        After: L.layer4.0.downsample_1

Reply:
I just added a test case that checks the module name after the name filtering is applied.
**test_inductor_extern_kernel_mod_name()**